gunicorn/util.py

```diff
@@ -54,6 +52,12 @@ def _setproctitle(title):
         pass
 
 
+def _load_entry_point(dist, section, name):
+    import pkg_resources
```

Comment:
Pretty sure it should be importlib_metadata` in 2022. Also, it doesn't have the slowness of `pkg_resources` AFAIR.

Reply:
Yes, pkg_resources not working well is a big reason why importlib.metadata is now in the standard library. However, given how little bandwidth the maintainers here seem to have, I'm going to keep this PR to something that obviously preserves behaviour so that it's easy to review.